src/common/snippets/src/op/subgraph.cpp

```diff
@@ -475,8 +475,8 @@ snippets::Schedule Subgraph::generate_from_linear_ir(const std::shared_ptr<lower
     LoweringResult lowering_result;
     control_flow_transformations(linear_ir, lowering_result, lowered_pass_config, backed_passes);
 #ifdef SNIPPETS_DEBUG_CAPS
-    if (linear_ir.get_config().perf_count_mode == lowered::PerfCountMode::Chrono) {
-        lowered::pass::InsertPerfCount perf_count_pass;
+    if (linear_ir.get_config().perf_count_mode != lowered::PerfCountMode::Disabled) {
```

Comment:
Could you please help me to understand what will happen if `PerfCountMode` is `BackendSpecific`?
As I understood, in this case we call common `InsertPerfCount` transformation which inserts common (std::chrono) perf counter nodes. So in case of `BackendSpecific` mode we will still use Chrono counters - this sounds strange for me.
Or here is the assumption that the backend should have a transformation that converts chrono counters into backend-specific ones?

Reply:
That's right.
The idea is that if we set `perf_count_mode` to `BackendSpecific`, then the backend should also provide transformation to convert Chrono counters to backend-specific ops. I didn't add this transformation in this PR for two reasons:
1) Current cpu perf counters support only serial mode, so this transformation should not be enabled automatically. Otherwise it may yield invalid data if used inappropriately. I think the best solution here is that a developer should enable this transformation manually.
2) Presently you need to rebuild the project to use perf counters, so the easiest way to enable backend-specific counters now - is to change the insert_perf_count pass directly (include necessary headers there, insert appropriate counter type, etc). This is a customized approach, but collecting perf counters via emitters is always done manually anyway.